lib/cb/clients/base.rb

```diff
@@ -14,6 +14,14 @@ class Base
       def self.cb_client
         @cb_client ||= Cb::Utils::Api.instance
       end
+
+      def self.headers(args)
+        {
+            'Accept' => 'application/json',
+            'Authorization' => "Bearer #{ args[:oauth_token] }",
```

Comment:
I agree with making it a little more generic. He did not use the token object because this is the public facing gem and anyone using this will not have the same context that we do.